src/components/AllJS/booklist.js

```diff
@@ -1,16 +1,26 @@
-import React from 'react';
-import { useSelector } from 'react-redux';
+import React, { useEffect } from 'react';
+import { useSelector, useDispatch } from 'react-redux';
 import AddBook from './addbook';
+import { getBooks } from '../../redux/books/books';
 
-const BookList = () => {
+function BookList() {
   const selector = useSelector((state) => state.bookReducer);
-
+  const dispatch = useDispatch();
+  useEffect(() => {
+    console.log(getBooks);
+    dispatch(getBooks());
+  }, []);
```

Comment:
- [ ] I think it would be better to remove all `console.log()` from your project  to keep your code clean and follow [the best practice](https://github.com/microverseinc/curriculum-html-css/blob/main/articles/javascript_best_practices.md)💯

![image](https://user-images.githubusercontent.com/45231276/204223105-d719076d-1c08-452b-8565-86160c1c1b98.png)